README.md

```diff
@@ -94,6 +94,13 @@ nvm is a version manager for [node.js](https://nodejs.org/en/), designed to be i
 ### Install & Update Script
 
 To **install** or **update** nvm, you should run the [install script][2]. To do that, you may either download and run the script manually, or use the following cURL or Wget command:
+
+If you are on a MAC and using a zsh terminal run this command first. 
```

Comment:
I think you mean "Mac", "MAC" is a networking address.
```suggestion
If you are on a Mac and using a zsh terminal run this command first. 
```

Reply:
No worries, it happens. If it wasn't duplicated elsewhere, i'd love to add it, but since it is, i think i'll close. If there's other improvements (that aren't specific to your company) i'd be happy to review them :-)